### ISSUE TITLE
Doc: Connecting via uart doesnt describe the connection to the DUT

### ISSUE THREAD

**Issue by fhajredini** [SamsungSLAV/muxpi#2]
Instead, that documentation describes the connection to muxPi board and nanoPi Neo. There is no clear description of the DUT connection. Not via ethernet, not via USB and nor UART. The documentation fails to document the main use case.

**Comment by amalinowsk2** [comment]
Yes, you're right, however, it is not that difficult to figure out how to do it. I believe that it is the most important thing to describe how to connect to MuxPi from a PC - without that you can't do anything.
All interfaces are described (e.g. DUT UART) or quite well-known (e.g. Ethernet) and if you know what you want you can for sure do it with MuxPi. 
Keep in mind that MuxPi is a device rather for advanced software/hardware developers than newbies.
And of course if you feel that you want to create and share such docs then don't hesitate to contribute.

**Comment by fhajredini** [comment]
I understand. I will try my best to improve the documentation.
